CHANGELOG.md

```diff
@@ -4,9 +4,32 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+
+## [0.1.4][] - 2022-08-15
+
+[0.1.4]: https://github.com/chaostoolkit-incubator/chaostoolkit-wiremock/compare/0.1.3...0.1.4
+
```

Comment:
Forgive me, I forgot to mention you shouldn't set the date/version in a PR. I do that during release.